apollos-church-api/src/data/index.js

```diff
@@ -4,12 +4,12 @@ import {
   createApolloServerConfig,
   Interfaces,
 } from '@apollosproject/server-core';
+import ApollosConfig from '@apollosproject/config';
 
 import * as Analytics from '@apollosproject/data-connector-analytics';
 import * as Scripture from '@apollosproject/data-connector-bible';
 import * as LiveStream from '@apollosproject/data-connector-church-online';
 import * as Cloudinary from '@apollosproject/data-connector-cloudinary';
-import * as OneSignal from '@apollosproject/data-connector-onesignal';
```

Comment:
@vinnyjth I wasn't sure about the one signal stuff including your new notification stuff, could use your guidance to make sure nothing breaks

Reply:
Yah this is tricky. A good reason to have two `index` files. I can pair with you on trying to come up with a one file solution for `OneSignal` though if we decide one `index` is better